# Other resources - metagenomics & stuff

Please keep in mind that there are tones of tools, some people might recommend different things...

You might want to look up papers from people working on the same type of samples as you are and who have been doing that for some time :)

## pull out SSU from metagenome
EMIRGE (widely used)
https://github.com/csmiller/EMIRGE

SortMeRNA (also widely used)
https://bioinfo.lifl.fr/RNA/sortmerna/

Phyloflash (new)
https://github.com/HRGV/phyloFlash

## PICRUSt2 
(Phylogenetic Investigation of Communities by Reconstruction of Unobserved States) is a software for predicting functional abundances based only on marker gene sequences.
https://github.com/picrust/picrust2/wiki

Mara (our own TA :) ) made a YouTube tuto on picrust 2
https://www.youtube.com/watch?v=xZ7yc-GKcSk

## BINNING
Recommend this read about standard for MAG from single cells or binned from metagenomes 
https://www.nature.com/articles/nbt.3893

Also this tutorial from the Meren lab if you are starting 
http://merenlab.org/tutorials/infant-gut/

Other binning software / or bin QC to consider
Metabat2 https://bitbucket.org/berkeleylab/metabat/src/master/
MaxBin2 https://sourceforge.net/projects/maxbin2/files/
DASTool https://github.com/cmks/DAS_Tool
MetaWrap (modular) https://github.com/bxlab/metaWRAP

## Kraken
https://github.com/DerrickWood/kraken2/wiki
Kraken 2 is the newest version of Kraken, a taxonomic classification system using exact k-mer matches to achieve high accuracy and fast classification speeds

## Kbase webinar
KBase: https://kbase.us/webinars/

## Intro to omics
Including explanation of kmers, read mapping and more!
https://www.youtube.com/channel/UCVFH6ULygyqqLGfDnmf0G4A

## HOPS 
Add-on after MALT to work on SAM files (alignements)
https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1903-0


## EukDetect
BioRXiv paper pulling out markers , trained on fungi, equivalent to HUMAnN2 but for fungi  
https://www.biorxiv.org/content/10.1101/2020.07.22.216580v1.full



